lib/aruba.py

```diff
@@ -94,6 +89,13 @@ def _TranslatePolicy(self, pol, exp_info):
       if 'ipv6' in filter_options:
         ip_ver = 6
       new_terms = []
+      for t in terms:
```

Comment:
i just added some comment support into this platform, it previously didn't support it. :(  iirc, another platform followed this pattern and I was just trying to stick to it.